synnax/pkg/api/ontology.go

```diff
@@ -180,7 +185,7 @@ func (o *OntologyService) AddChildren(
 ) (res types.Nil, err error) {
 	if err = o.access.Enforce(ctx, access.Request{
 		Subject: getSubject(ctx),
-		Action:  access.Create,
+		Action:  ontology.AddChildrenAction,
```

Comment:
Not against these having their own types, just wondering - why this can't be a generic `create` action instead of having a specific action type?